specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspAssociationDelete.json

```diff
@@ -9,6 +9,7 @@
   },
   "responses": {
     "200": {},
+    "202": {},
```

Comment:
It has to return `Location` header for long running 202 response.

Reply:
Added location header